node/src/network.rs

```diff
@@ -0,0 +1,116 @@
+    Arabica,
+    Mocha,
+    #[default]
+    Private,
+}
+
+#[derive(Debug, Error)]
+#[error("unknown network {0}")]
+pub struct UnknownNetworkError(String);
+
+impl FromStr for Network {
+    type Err = UnknownNetworkError;
+
+    fn from_str(network_id: &str) -> Result<Self, Self::Err> {
+        match network_id {
+            "arabica-10" => Ok(Network::Arabica),
+            "mocha-4" => Ok(Network::Mocha),
+            "private" => Ok(Network::Private),
+            network => Err(UnknownNetworkError(network.to_string())),
+        }
+    }
+}
+
+pub fn network_id(network: Network) -> &'static str {
+    match network {
+        Network::Arabica => "arabica-10",
+        Network::Mocha => "mocha-4",
+        Network::Private => "private",
+    }
+}
+
+pub fn network_genesis(network: Network) -> Option<Hash> {
+    let hex = match network {
+        Network::Arabica => "5904E55478BA4B3002EE885621E007A2A6A2399662841912219AECD5D5CBE393",
+        Network::Mocha => "B93BBE20A0FBFDF955811B6420F8433904664D45DB4BF51022BE4200C1A1680D",
+        Network::Private => return None,
+    };
+
+    let bytes = hex::decode(hex).expect("failed decoding genesis hash");
+    let array = bytes.try_into().expect("invalid genesis hash lenght");
+
+    Some(Hash::Sha256(array))
+}
+
+pub fn canonical_network_bootnodes(network: Network) -> Vec<Multiaddr> {
+    match network {
+        Network::Arabica => [
+                "/dns4/da-bridge.celestia-arabica-10.com/tcp/2121/p2p/12D3KooWM3e9MWtyc8GkP8QRt74Riu17QuhGfZMytB2vq5NwkWAu",
+                "/dns4/da-bridge-2.celestia-arabica-10.com/tcp/2121/p2p/12D3KooWKj8mcdiBGxQRe1jqhaMnh2tGoC3rPDmr5UH2q8H4WA9M",
+                "/dns4/da-full-1.celestia-arabica-10.com/tcp/2121/p2p/12D3KooWBWkgmN7kmJSFovVrCjkeG47FkLGq7yEwJ2kEqNKCsBYk",
+                "/dns4/da-full-2.celestia-arabica-10.com/tcp/2121/p2p/12D3KooWRByRF67a2kVM2j4MP5Po3jgTw7H2iL2Spu8aUwPkrRfP",
+            ]
+            .iter()
+            .map(|s| s.parse())
+            .collect()
+        ,
+        Network::Mocha => [
+            // 40.85.94.176 is a node set up for testing QUIC/WebTransport since official nodes
+            // don't have that enabled currently
+            "/ip4/40.85.94.176/tcp/2121/p2p/12D3KooWNJ3Nf1DTQTz8JZogg2eSvPKKKv8itC6fxxspe4C6bizs",
+            "/ip4/40.85.94.176/udp/2121/quic-v1/p2p/12D3KooWNJ3Nf1DTQTz8JZogg2eSvPKKKv8itC6fxxspe4C6bizs",
+            "/ip4/40.85.94.176/udp/2121/quic-v1/webtransport/certhash/uEiBf-OX4HzFK9owOpjdCifsDIWRO0SoD3j3vGKlq0pAXKw/certhash/uEiCx1md1BATJ_0NXAjp3KOuwRYG1535E7kUzFdMq8aPaWw/p2p/12D3KooWNJ3Nf1DTQTz8JZogg2eSvPKKKv8itC6fxxspe4C6bizs",
+            "/ip4/40.85.94.176/udp/2121/quic-v1/p2p/12D3KooWQUYAApYb4DJnhS1QmAwRr5HRvUeHJYocchCpwEhCtDGu",
+            "/ip4/40.85.94.176/udp/2121/quic-v1/webtransport/certhash/uEiBr4-sr95BpqfA-ttpjiLdjbGABhTvX8oxrTXf3Ubfibw/certhash/uEiBSVgyze9xG1UbbNuTwyEUWLPq7l2N9pyeQSs3OtEhGRg/p2p/12D3KooWQUYAApYb4DJnhS1QmAwRr5HRvUeHJYocchCpwEhCtDGu",
```

Comment:
These need to be updated every two weeks. I'm suggesting the following (it can be done in another task):

Implement some kind of lookup function that will return all the addresses of the specified node. The `serve_config` will use the TCP multiaddr to fetch the addresses and serve the WebTransport addresses via `/cfg.json`.

Reply:
> /dns4/celestia1.nodes.eiger.co/udp/2121/quic-v1/webtransport/p2p/12D3KooW9z4jLqwodwNRcSa5qgcSgtJ13kN7CYLcwZQjPRYodqWx

please use this one for now